cycle_for/task_06.py

```diff
@@ -0,0 +1,9 @@
+# По данному натуральному n вычислите сумму 13+23+33+...+n3.
+
+n = int(input())
+
+sum = 0
+for i in range(1, n + 1):
```

Comment:
I don't think the solution is correct. Does it pass all the tests?

Reply:
What's wrong?

Reply:
@Shillerm I see now. From the comments, I thought you were supposed to calculate 13 + 23 + ... n3 but it's actually the cube - 1^3 + 2 ^3 +...n^3. I checked the question in the site so now I know it is correct